tensorflow/core/kernels/mkl_pooling_ops_common.h

```diff
@@ -280,6 +280,8 @@ class MklPoolingBwdPrimitive : public MklPrimitive {
     std::shared_ptr<mkldnn::memory> diff_dst_mem;
 
     // Memory descriptors.
+    std::shared_ptr<mkldnn::memory::desc> src_md;
+    std::shared_ptr<mkldnn::memory::desc> dst_md;
     std::shared_ptr<mkldnn::memory::desc> diff_src_md;
     std::shared_ptr<mkldnn::memory::desc> diff_dst_md;
```

Comment:
Could you please add a comment that these are only used in v0 code and should be cleaned up when v0 is removed?

Reply:
Hi Penporn, this issue does not happen with DNN v0.x build. I have sent a message to Yang to add a comment as you suggestion. 

BTW, v0 code clean-up will be in either TF 2.4 or 2.5. We may prefer to keeping v0 integration code for TF 2.4 as it will help to future trouble-shooting work. Thanks!

Reply:
We will tell you when we decide to clean up v0 code, as it will generate lots of PR's ...

Reply:
@gzmkl Thank you for the quick answer! I understand that v0 removal will take time and a lot of PR, and I don't mean to rush the removal (please take your time). Just wanted to mark these variables because they are not explicitly wrapped in `#ifdef`s and might be harder to spot during the clean-ups. :)

Now that I think of it, maybe we should wrap them in an `#ifdef`. Up to @ShengYang1 then.

Reply:
`#ifdef` added, and also cleaned some v1 code to not use `diff_src_md`.

Reply:
Thank you for the clean up!